src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -41,21 +46,21 @@ public void execute_listIsNotFiltered_showsSameList() {
     }
 
     @Test
-    public void execute_listIsFiltered_showsLectures() {
+    public void execute_listIsFiltered_moduleFoundShowLectures() {
         ModuleCode moduleCode = module.getCode();
         String input = String.format("list /mod %s", moduleCode);
         try {
             listCommand = new ListCommandParser().parse(input);
             listCommand.execute(expectedModel);
```

Comment:
Not sure if it's a good idea to use the `execute` method to set up the `expectedModel`. Using the SUT to set up the expected result of the test might let bugs slip through if the SUT is bugged. Perhaps, setup `expectedModel` using methods of `Model`